### PR TITLE
Fix for UnicodeDecodeError: ascii codec cannot decode byte while saving a TIFF image

### PR DESCRIPTION
**Pull request by bogdan199**
Problem occured while saving TIFF images that contain non-ascii characters in metadata.

Solves issue #272

### CODE REVIEW

**Comment by aclark4life** [comment]
screenshot 2015-07-02 05 23 55

**Comment by wiredfool** [comment]
IIRC from the tiff spec, the metadata needs to be ascii values < 127. Arguably, erroring is the right thing, esp if we have an explanation handy so that the user can do reasonable substitutions, rather than just dropping the character.

**Comment by mkubala** [comment]
@wiredfool you're right, in general approach of 'explicitly signaled failure' good, except the case of batch  processing when you have a lot of (not even dozens but thousands and more) TIFF images with broken metadata encoding. Fix submitted by @bogdan199 makes hard failures recoverable, I like it!

**Comment by bogdan199** [comment]
I've just resolved all conflicts. 
Looking forward for a review.

File: PIL/TiffImagePlugin.py [hunk 1]
```diff
@@ -549,7 +549,9 @@ def save(self, fp):
                 # and doesn't match the tiff spec: 8-bit byte that
                 # contains a 7-bit ASCII code; the last byte must be
                 # NUL (binary zero). Also, I don't think this was well
-                # exercised before.
+                # excersized before.
```
Comment (hugovk) [inline PIL/TiffImagePlugin.py]: "excersized" isn't a word in US or UK English.

Comment (radarhere) [inline PIL/TiffImagePlugin.py]: Looks like the conflict was resolved the wrong way, reversing the effect of PR #1311

**Comment by wiredfool** [comment]
See #1416

**Pull request closed without merging**